### ISSUE TITLE
2.0.0-dev* npm packages not published as preview

### ISSUE THREAD

**Issue by greg-murray-volusion** [artilleryio/artillery#943]
```bash
npm view artillery

artillery@2.0.0-dev2 | MPL-2.0 | deps: 40 | versions: 83
...
dist-tags:
latest: 2.0.0-dev2
```

**Comment by hassy** [comment]
Thank you @greg-murray-volusion! Should be fixed now, with `latest` = `1.6.2` and `beta` = `2.0.0-dev1`

**Comment by greg-murray-volusion** [comment]
thanks for the quick response!
